package.json

```diff
@@ -96,6 +96,8 @@
     "graphql": "^15.5.1",
     "husky": "^6.0.0",
     "jest": "^27.0.5",
+    "jsonwebtoken": "^8.5.1",
+    "msw": "^0.35.0",
```

Comment:
Can we remove pollyjs from dependencies?

Reply:
We can't, we still use polly in user.test.ts